resources/user-guides/signal-extension-and-rules-engine-integration.md

```diff
@@ -12,11 +12,7 @@ To send PII data to external destinations, the `PII` action can trigger the Rule
 
 ## Rules tokens <a id="rules-tokens"></a>
 
-Rules tokens are special strings that are used in rule actions as values and are expanded by the SDK when the action is carried out. The format of a token is \`
-
-`, where token is any data element that is defined in Adobe Experience Platform Launch for a mobile property that identifies the source of the data from which the token is expanded. For example,`
-
-`can be used in the Signal postback action, where`My Data element for ECID\` is a data element that was created using the Mobile Core extension, and the data element type is Experience Cloud ID.
+Rules tokens are special strings that are used in rule actions as values and are expanded by the SDK when the action is carried out. The format of a token is `{%%token%%}`, where token is any data element that is defined in Adobe Experience Platform Launch for a mobile property that identifies the source of the data from which the token is expanded. For example, `{%%My Data element for ECID%%}` can be used in the Signal postback action, where `My Data element for ECID` is a data element that was created using the Mobile Core extension, and the data element type is Experience Cloud ID.
```

Comment:
Please change "Adobe Experience Platform Launch" to "Experience Platform Launch". 
(We only need to have the complete name at the 1st mention, and I'm sure it's been mentioned already!)

Reply:
Kept this one the same and renamed the next occurrence.